CONTRIBUTING.rst

```diff
@@ -65,23 +66,22 @@ include:
   to the code you are scanning.
 * The errors messages or failure trace if any.
 * If helpful, you can add a screenshot as an issue attachment when relevant or
-  some extra file as a link to a Gist https://gist.github.com
+  some extra file as a link to a `Gist <https://gist.github.com>`_
 
 
 Documentation improvements
 ==========================
 
 Documentation can come in the form of wiki pages, docstrings, blog posts,
-articles, etc. Even a minor typo fix is welcomed.
+articles, etc. Even a minor typo fix is welcomed. A start can be given with `Pages of Wiki <https://github.com/nexB/scancode-toolkit/wiki>`_.
 
```

Comment:
I am not sure I get what your meant in this sentence?

Reply:
It was just reference to say `here is something to update`, but I forgot the fact that it's for the new comers. But remove it, it's not necessary at all. 😄